src/sentry/replays/usecases/query/__init__.py

```diff
@@ -163,6 +165,24 @@ def query_using_optimized_search(
             SearchFilter(SearchKey("environment"), "IN", SearchValue(environments)),
         ]
 
+    for i in range(len(search_filters)):
+        search_filter = search_filters[i]
+        if (
+            isinstance(search_filter, SearchFilter)
+            and search_filter.key.name in VIEWED_BY_ME_KEY_ALIASES
+        ):
+            # "viewed_by_me" is not a valid query field.
+            # It's a convenience alias for users without the admin access to lookup ids.
```

Comment:
>            # It's a convenience alias for users without the admin access to lookup ids.

isn't this all users? should we just remove this comment as no sentry users have admin access?

Reply:
Yeah, maybe
"It's a convenience alias since only Sentry employees can lookup ids"
?

Reply:
why do we need to clarify this?

Reply:
(it seems to be a given)

Reply:
Other ppl on our team weren't clear about this when we built it, which is why we exposed viewed_by_id

Reply:
right, but the comment implies that us going and looking up ids is a normal use case, but it is not -- i think thats where my confusion lies

Reply:
Oh, I see. I guess that can be misleading, I can remove it then